From 3121ba11360e21acbd6e20cda01eb8124c077c7b Mon Sep 17 00:00:00 2001
From: Tiejun Chen <tiejun.chen@windriver.com>
Date: Tue, 4 Dec 2012 13:16:52 +0800
Subject: [PATCH 02/22] powerpc/fsl-pci: Add p2041_rdb and p3060_qds support

Signed-off-by: Jia Hongtao <B38951@freescale.com>
[Extracted from QorIQ-SDK-V1.3-SOURCE-20121114-yocto.iso but
only keep p2041_rdb specific stuff.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/platforms/85xx/p2041_rdb.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/platforms/85xx/p2041_rdb.c b/arch/powerpc/platforms/85xx/p2041_rdb.c
index 66e8123..d83ef2e 100644
--- a/arch/powerpc/platforms/85xx/p2041_rdb.c
+++ b/arch/powerpc/platforms/85xx/p2041_rdb.c
@@ -81,6 +81,7 @@ define_machine(p2041_rdb) {
 	.init_early		= corenet_ds_init_early,
 };
 
+machine_arch_initcall(p2041_rdb, corenet_ds_publish_pci_device);
 machine_device_initcall(p2041_rdb, corenet_ds_publish_devices);
 
 #ifdef CONFIG_SWIOTLB
-- 
1.7.9.7

